### ISSUE TITLE
Getting same top 10 word for each iteration on 20-Newsgroup dataset

### ISSUE THREAD

**Issue by durgeshbhagat** [nateraw/Lda2vec-Tensorflow#61]
While running the lda2vec on 20-Newsgroup downloaded from sklearn datasets, I am getting same top 10 word for each iteration.

`Output` : 
EPOCH: 2 time taken: 42.41943836212158,  LOSS 4.823064 w2v 4.823064 lda 79421.92
---------Closest 10 words to .............................given indexes----------
Topic 0 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, etc, addition
Topic 1 : likewise, unfortunately, instance, furthermore, example, presumably, similarly, actually, etc, addition
Topic 2 : likewise, instance, unfortunately, furthermore, example, presumably, similarly, actually, etc, addition
Topic 3 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 4 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 5 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 6 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 7 : likewise, unfortunately, instance, furthermore, example, presumably, similarly, actually, etc, addition
Topic 8 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 9 : likewise, unfortunately, instance, furthermore, example, presumably, similarly, actually, addition, etc
Topic 10 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 11 : likewise, unfortunately, instance, furthermore, example, presumably, similarly, actually, etc, addition
Topic 12 : likewise, unfortunately, instance, furthermore, example, presumably, similarly, actually, etc, addition
Topic 13 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 14 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 15 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 16 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 17 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 18 : likewise, unfortunately, instance, furthermore, presumably, example, similarly, actually, supposedly, addition
Topic 19 : likewise, instance, unfortunately, furthermore, example, presumably, similarly, actually, etc, addition

Would highly appreciate any help on it?

Thanking You,
Durgesh Kumar

**Comment by dbl001** [comment]
Hi Durgesh,

The first 5 Epoch are for training the word vectors not the topic vectors (unless you changed):

# Epoch that we want to "switch on" LDA loss
switch_loss_epoch = 5


Please try running for at least 25 Epochs (200 is recommended).
